Tests/Shared.Specs/CollectionEquivalencySpecs.cs

```diff
@@ -900,14 +900,45 @@ public void When_all_subject_items_are_not_equivalent_to_expectation_object_it_s
             // Act
             //-----------------------------------------------------------------------------------------------------------
             Action action = () => subject.Should().AllBeEquivalentTo(1);
-
+            
             ////-----------------------------------------------------------------------------------------------------------
             //// Assert
             ////-----------------------------------------------------------------------------------------------------------
             action.Should().Throw<XunitException>().WithMessage(
                 "Expected item[1] to be 1, but found 2.*Expected item[2] to be 1, but found 3*");
         }
 
+        [Fact]
+        public void When_some_subject_items_are_not_equivalent_to_expectation_for_huge_table_execution_time_should_still_be_short()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            const int N = 100000;
```

Comment:
🔧 How about using `Enumerable.Range(1, 100_000).ToList()`?

Reply:
Notice that first item is not equal to the rest. If the test will ultimately use single value I will refactor to `Enumerable.Range`.